infra.lvn_tools/extend_boot/galaxy.yml

```diff
@@ -0,0 +1,69 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: infra.lvn_tools
```

Comment:
@swapdisk is this the correct namespace?

Reply:
infra.lvm_tools

Reply:
Renamed to `infra` as agreed.